app/components/appeals-app-services/monitoring.tf

```diff
@@ -0,0 +1,27 @@
+# TODO: Add monitoring for function apps, specific to Appeals

```

Comment:
Do we need this file? Just not a fan of commented out things hanging around the code unless they're going to be re-enabled exactly as they are. Feel like this one won't be?

Reply:
Removed in https://github.com/Planning-Inspectorate/infrastructure-environments/pull/49/commits/e7e087e9875a1f7e8117b93796466072eeb4be3c and will add Appeals-specific monitoring in a later PR.